src/main/java/Duke.java

```diff
@@ -1,10 +1,71 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+import java.util.*;
+
+public class Duke{
+    Scanner sc = new Scanner(System.in);
+    Task[] t = new Task[100];
+    private static int taskNo = 0;
+
+    public void run() {
+
+        String input;
+        String[] switchChoice;
+        do {
+            Ui.printMenu();
+            input = sc.nextLine().trim().toLowerCase();
+            switchChoice = input.toLowerCase().split(" ", 2);
+            switch (switchChoice[0]){
+            case "todo":
+                enterToDo(switchChoice[1]);
+                break;
+            case "deadline":
+                enterDeadline(switchChoice[1]);
+                break;
+            case "event":
+                enterEvent(switchChoice[1]);
+                break;
+            case "list":
+               printDescription();
+                break;
+            case "done":
+                markDone(switchChoice[1]);
+                break;
+            }
+        } while (!input.equals("bye"));
+    }
+
+
+    private void enterDeadline(String input) {
```

Comment:
I like the way that all 3 methods are of the same naming convention to show that they are doing similar tasks. But I think the naming can be changed to make it slightly clearer? Like "addDeadline" etc. as enter might be misleading depending on subsequent functions created to key in deadlines. :)